Static/1_Sentence.sql

```diff
@@ -83,7 +83,7 @@ INSERT INTO Sentence (id,culture,value,length) VALUES (92,1033,'Retained Earning
 INSERT INTO Sentence (id,culture,value,length) VALUES (92,1036,'Des bénéfices non répartis',26);
 INSERT INTO Sentence (id,culture,value,length) VALUES (92,2058,'Ganancias retenidas',19);
 INSERT INTO Sentence (id,culture,value,length) VALUES (93,1033,'Discounts',9);
-INSERT INTO Sentence (id,culture,value,length) VALUES (93,1036,'l' escompte',11);
+INSERT INTO Sentence (id,culture,value,length) VALUES (93,1036,'l'' escompte',11);
```

Comment:
Is there a reason that you added an extra ' here?

Reply:
At least for PostgreSQL, a single quote is escaped in a string by two in a row.  Curious if another character, other than a single quote is valid?

Reply:
@jazd I believe with postgres, the two single quotes is the correct way to escape in a string. Double quotes are a delimiter identifier.